dev-php/pecl-event/pecl-event-2.5.3.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PHP_EXT_NAME="event"
+PHP_EXT_INI="yes"
+PHP_EXT_ZENDEXT="no"
+DOCS=( README.md )
+
+USE_PHP="php5-6 php7-1 php7-2 php7-3"
```

Comment:
I think you should drop php-5.6 support, it should be getting removed soon from the Gentoo tree.